dev-libs/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/$portName-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 0
+	libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:plistutil$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+	
+PROVIDES_devel="
+	libplist${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	devel:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	libxml2$secondaryArchSuffix
+	"
+	
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:python
+	cmd:pkg_config
+	cmd:sh
+	"
+	
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+INSTALL()
+{
+	make install
+
+	# remove libtool file
+	rm $libDir/$portName++.la
```

Comment:
Oops, if there is also a **`libplist.la`** (and I guess this is the case), then you will probably want to use:
```
	rm $libDir/libplist*.la
```
or
```
	rm $libDir/lib*.la
```
(Both suggestions are OK, so you can choose the one you prefer ;-)